src/style.css

```diff
@@ -0,0 +1,132 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 198px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  margin: 32px 19px 40px;
+  height: 134px;
+  width: 160px;
+}
+
+.card__name-of-product {
+  margin: 0 16px 4px;
+
+  font-family: inherit;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code-of-product {
+  margin-left: 16px;
+
+  font-family: inherit;
+  font-weight: 400;
+  font-size: 10px;
+
+  color: #616070;
+}
+
+.card__reviews {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-top: 16px;
+  height: 16px;
+}
+
+.card__reviews-stars {
+  display: flex;
+  margin-left: 16px;
+  text-align: center;
+}
+
+.star {
+  display: flex;
+  background: url(./images/star.svg) no-repeat;
+
+  height: 16px;
+  width: 16px;
+}
+
+.star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.card__reviews-stars > :nth-child(-n + 4) {
+  background: url(./images/star-active.svg) no-repeat;
+}
+
+.card__reviews-text {
+  margin-right: 16px;
+  font-family: inherit;
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 24px 16px 16px;
+  color: #060b35;
+}
```

Comment:
[STYLES] - Be consistent with your `margins`. Add only top or bottom, don't add both.